doc/proposals/cli-ux-phase1.md

```diff
@@ -0,0 +1,123 @@
+# User Experience CLI Improvements | Phase 1
+
+Implementation Owner: @joelanford
+
+Status: Draft
+
+[Background](#Background)
+
+[Goal](#Goal)
+
+[Use cases](#Use_cases)
+
+[Proposed CLI commands](#Proposed_CLI_commands)
+
+[References](#References)
+
+## Background
+
+The SDK CLI is one of the primary tools in the Operator Framework for operator developers and
+deployers. However, the current user experience (UX) leaves a lot to be desired. Currently, the
+tools and documentation that help a user create, develop, test, package, run, and publish an
+operator are spread among many different repositories, making for a steep learning curve for 
+newcomers.
+
+## Goal
+
+Describe an initial set of CLI additions to support a simplified workflow for an Operator SDK user to run an Operator SDK-developed project in a cluster using OLM.
```

Comment:
Nit:
Maybe we can rephrase the word describe, as part of this proposal goal is to also implement those CLI commands. This just sounds like to me at least, that we want to just describe them :) As I said, it's a nit. :)